pkg/api/aim2/dao/dto/metric.go

```diff
@@ -0,0 +1,10 @@
+package dto
```

Comment:
Can this just be part of models.Metric? I don't think we should add another package type

Reply:
it seems like this is unique key rather than a transfer object, so i think the package name is not helping to explain what this is. why not have models.MetricKey?